web/src/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
My bad, I moved, made changes then committed - this is mostly the same except I had to add the index.ts line + I removed `%PUBLIC_URL%.